nifi-docker/dockermaven/pom.xml

```diff
@@ -44,7 +44,6 @@
                                     </buildArgs>
                                     <repository>apache/nifi</repository>
                                     <tag>${project.version}</tag>
-                                    <tag>latest</tag>
```

Comment:
Why was this removed? When we do a release, I believe we want to keep that release as the latest.

Reply:
I removed it because it did not build both tags, just the last one, so I had a choice. Also this is only a dev/test config and does not affect the public image.

Reply:
Understood. Thanks.